.github/workflows/main.yml

```diff
@@ -30,11 +30,14 @@ jobs:
     - name: Set up Ruby
       uses: ruby/setup-ruby@v1
       with:
-        ruby-version: "2.5"
```

Comment:
This needs to be 2.5 and cannot change because of CRuby's workflow for packaging binaries.